src/rt/sched/backpressure.h

```diff
@@ -166,7 +166,7 @@ namespace verona::rt
      */
     uint32_t _has_token : 1;
     /**
-     * Indicates if a cown is currently muted (1), unmutable (2), or
+     * Indicates if a cown is currently normal (0), muted (1), unmutable (2), or
```

Comment:
nit: `normal` can mean many things, easily overloaded term. Perhaps `unmuted` would be better in this context?

Reply:
Normal is the term used in the big comment at the top of the file. It will be rewritten soon, so I'll make a note to myself to come up with a better name.